config/samples/backends/hpe/3par/fc/backend.yaml

```diff
@@ -0,0 +1,33 @@
+# To be able to use this sample it is necessary to:
+# - Have a 3PAR backend with FC support
+# - Have the 3PAR credentials in cinder-volume-3par-secrets.yaml
+# - Having multpathd running on the host (done by this sample)
+# - Build a cinder-volume container with the 3parclient cinder driver dependency, make it available in a registry accessible by OpenShift, and set its location in this file's containerImage
+#
+# There is a sample Dockerfile in the parent directory showing how the cinder-volume container in this containerImage was created.
+# The correct way to configure and run the daemons is using `MachineConfig` as shown in `iscsid.yaml` and `multipathd.yaml`, like this sample does.
+#
+# ATTENTION: After applying this OpenShift nodes will reboot, because of the `MachineConfig` changes and will take a while to recover.
+
+apiVersion: core.openstack.org/v1beta1
+kind: OpenStackControlPlane
+metadata:
+  name: openstack
+spec:
+  cinder:
+    template:
+      cinderVolumes:
+        hpe3par-fc:
+          networkAttachments:
+          - storage
+          containerImage: quay.io/geguileo/podified-antelope-openstack-cinder-volume-3par:current-podified
+          customServiceConfigSecrets:
+            - cinder-volume-3par-secrets
+          customServiceConfig: |
+            [hpe3par]
```

Comment:
Given that we're offering both FC and iscsi examples, should the backend section (and perhaps the volume_backend_name) distinguish between them, as you do in the CR name given on L20?

That reminds me: the documentation should emphasize the examples are designed to be applied one at a time, and you can't get two backends by applying a second one. Each sample you apply will cause an existing c-vol backend to be removed.

Reply:
I didn't distinguish it for any of the other vendor samples either, and I think it's not worth it.  In my own deployments I don't usually distinguish them when I only have 1 transport protocol from the array.

I have added a new paragraph on the README.md file about the single sample thingy.